crates/astria-composer/tests/blackbox/helper/mock_sequencer.rs

```diff
@@ -57,3 +59,62 @@ pub async fn mount_abci_query_mock(
         .mount_as_scoped(server)
         .await
 }
+
+async fn mount_cometbft_status_response(
+    server: &MockServer,
+    mock_sequencer_chain_id: &str,
+) -> MockGuard {
+    let mut status_response: status::Response = serde_json::from_str(STATUS_RESPONSE).unwrap();
+    status_response.node_info.network = mock_sequencer_chain_id.to_string().parse().unwrap();
+
+    let response =
+        tendermint_rpc::response::Wrapper::new_with_id(Id::Num(1), Some(status_response), None);
+
+    Mock::given(body_partial_json(json!({"method": "status"})))
+        .respond_with(ResponseTemplate::new(200).set_body_json(response))
+        .up_to_n_times(1)
+        .expect(1..)
+        .named("CometBFT status")
+        .mount_as_scoped(server)
+        .await
+}
+
+const STATUS_RESPONSE: &str = r#"
```

Comment:
Please construct this object ad-hoc instead of a fixed string.

Reply:
Since status::Response has no default configuration, I'm not sure how to get around doing an fixed declaration of the response somewhere. As of now, I've just moved the construction of the response inside the mount function. Also worth noting that this is copied from the status response mount for the sequencer relayer, so any change made here may need to be reflected there as well. Do you have a preferred solution in mind for the fixed strings?

Reply:
You just have to manually fill in the blanks. :)

If you use the genesis RPC instead of status (which I think is more reasonable), then you can just copy what we have in conductor:
https://github.com/astriaorg/astria/blob/c31c10b6db41b19fdce8e4b172363f855d88be23/crates/astria-conductor/tests/blackbox/helpers/mod.rs#L287

Reply:
Should be resolved in #[f3c96ff]. Thanks for all the feedback :)